container/root/etc/services-available/php-fpm/.command

```diff
@@ -3,4 +3,13 @@
 # Ensures that a failure of php-fpm doesn't return with a 0, even if the right-hand side does
 set -o pipefail
 
-exec php-fpm -F -O 2>&1 | sed -u 's,.*: \"\(.*\)$,\1,'| sed -u 's,\"$,,'
+# IMPORTANT: PHP 7.3+ optionally allows undecorated stdout/stderr, removing pipe magic requirement
+UPGRADE_COMMAND=`grep "decorate_workers_output = no" $CONF_FPMPOOL`
```

Comment:
this is probably what makes it backwards compatible?

Reply:
exactly. the `ini` directive only exists where valid (7.3 and on)